src/ImageSharp.Formats.Png/PngEncoderCore.cs

```diff
@@ -488,18 +479,18 @@ private void WriteHeaderChunk(Stream stream, PngHeader header)
         private QuantizedImage<TColor> WritePaletteChunk<TColor>(Stream stream, PngHeader header, ImageBase<TColor> image)
             where TColor : struct, IPixel<TColor>
         {
-            if (this.Quality > 256)
+            if (this.quality > 256)
             {
                 return null;
             }
 
-            if (this.Quantizer == null)
+            if (this.quantizer == null)
```

Comment:
It looks like a mistake that I forgot to use the quantizer from the options. Thanks @antonfirsov! I could not set it inside the options due to the <TColor> restriction.